test/store.test.js

```
@@ -128,53 +128,13 @@ describe('Store API using inMemoryStorage', () => {
 
 describe('Store API bad interface', () => {
   let store = null
-  beforeAll(async () => {
-    store = new Store.Store('123', {})
-  })
-  it('should not init', async () => {
-    expect.assertions(1)
-    await store.init().catch(e => {
-      expect(e.name).toEqual(common.ERRORS.FUNCTIONNOTDEFINED)
-    })
-  })
-  it('should set a key', async () => {
```

Where have all these tests gone?

---

Because I considered that if the storage interface is not correct, the constructor should raise an error. Then obviously the user should not be able to use setItem, getItem and other operations. Furthermore, this test was testing that setting a key fails, its name is not reflecting the test). What do you think ?

---

I agree that the name didn't reflect the outcome. It doesn't change that all the rest that should fail are gone. :)